backend/database/seeders/PrintShopDemoSeeder.php

```
@@ -467,41 +471,77 @@ private function seedWorkOrders(array $pt, array $lines): void
+                        $status = match (true) {
+                            $weekOffset < 0 => WorkOrder::STATUS_DONE,
+                            $weekOffset > 0 => mt_rand(0, 1) ? WorkOrder::STATUS_PENDING : WorkOrder::STATUS_ACCEPTED,
+                            default => [
+                                WorkOrder::STATUS_IN_PROGRESS,
+                                WorkOrder::STATUS_IN_PROGRESS,
+                                WorkOrder::STATUS_ACCEPTED,
+                                WorkOrder::STATUS_PENDING,
+                            ][mt_rand(0, 3)],
+                        };
+
+                        WorkOrder::updateOrCreate(
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\WorkOrder::updateOrCreate().


(staticMethod.notFound)